core/main.go

```diff
@@ -86,7 +86,7 @@ func initDependencies() (*dependencies, error) {
 
 func deployCoreServices(config *config.Config, sdk *sdk.SDK) error {
 	for _, serviceConfig := range config.SystemServices {
-		logrus.Infof("Deploying service %q", serviceConfig.Definition.Sid)
+		logrus.WithField("module", "main").Infof("Deploying service %q", serviceConfig.Definition.Sid)
```

Comment:
It could be shortened by something like:
```go
var (
	logger = logrus.WithField("module", "main")
)
```
```suggestion
		logger.Infof("Deploying service %q", serviceConfig.Definition.Sid)
```

Reply:
I can do this when cosmos will be integrated and then will add also with filed in main.go (the one put when dev cosmos branch)